addons/mail/static/src/discuss/call/public_web/discuss_sidebar_call_participants.xml

```diff
@@ -2,53 +2,32 @@
-                <i class="oi o-xxsmaller text-muted me-1" t-att-class="{'oi-chevron-right': !state.expanded, 'oi-chevron-down': state.expanded}" t-att-title="title"/>
-            </button>
-            <AvatarStack
-                t-if="!state.expanded"
-                direction="compact ? 'v' : 'h'"
-                avatarClass="(p) => this.avatarClass(p)"
-                max="compact ? 1 : 4"
-                personas="sessions.map((s) => s.channel_member_id.persona)"
+            <div 
+                class="o-mail-DiscussSidebarCallParticipants d-flex py-1 bg-inherit rounded-3 ps-4 pe-2 px-2"
+                t-att-class="{ 'opacity-75': props.thread.notEq(rtc.state.channel) }"
+                t-ref="root"
             >
-                <t t-set-slot="avatarExtraInfo" t-slot-scope="scope">
-                    <div class="o-mail-DiscussSidebarCallParticipants-status small" t-att-class="{
-                        'position-absolute bg-inherit p-0 rounded-circle o-compact d-flex o-xsmaller text-start': compact,
-                        'ms-1 d-flex align-items-center justify-content-center': !compact,
-                    }">
-                        <t t-call="mail.DiscussSidebarCallParticipants.participantShortStatus">
-                            <t t-set="session" t-value="scope.persona.currentRtcSession"/>
-                        </t>
-                    </div>
-                </t>
-            </AvatarStack>
-            <t t-else="" t-call="mail.DiscussSidebarCallParticipants.list">
-                <t t-set="isCompact" t-value="compact"/>
-            </t>
-        </div>
+                <button class="btn btn-link p-0 align-self-start">
+                    <i class="oi o-xxsmaller text-muted me-1" t-att-class="{'oi-chevron-right': !state.expanded, 'oi-chevron-down': state.expanded}" t-att-title="title" t-on-click="() => state.expanded = !state.expanded"/>
+                </button>
+                <AvatarStack
+                    t-if="!state.expanded"
+                    direction="'h'"
+                    avatarClass="(p) => this.avatarClass(p)"
+                    max="4"
+                    personas="sessions.map((s) => s.channel_member_id.persona)"
+                >
+                    <t t-set-slot="avatarExtraInfo" t-slot-scope="scope">
+                        <div class="d-flex o-mail-DiscussSidebarCallParticipants-status ms-1 small align-items-center justify-content-center">
+                            <t t-call="mail.DiscussSidebarCallParticipants.participantShortStatus">
+                                <t t-set="session" t-value="scope.persona.currentRtcSession"/>
+                            </t>
+                        </div>
+                    </t>
+                </AvatarStack>
+                <t t-else="" t-call="mail.DiscussSidebarCallParticipants.list" />
+            </div>
```

Comment:
I remember making a recent PR to fix positioning of avatar stack on collapse/expand.
This is reverted for no reason again.